### PR TITLE
test: update ceph status output file in external script

### PR DESCRIPTION
**Pull request by subhamkrai**
Update ceph status output file in external script

Signed-off-by: subhamkrai <srai@redhat.com>

**Description of your changes:**

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/development-flow.html#commit-structure).
- [ ] **Skip Tests for Docs**: Add the flag for skipping the build if this is only a documentation change. See [here](https://github.com/rook/rook/blob/master/INSTALL.md#skip-ci) for the flag.
- [ ] **Skip Unrelated Tests**: Add a flag to run tests for a specific storage provider. See [test options](https://github.com/rook/rook/blob/master/INSTALL.md#test-storage-provider).
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/development-flow.html#submitting-a-pull-request)
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.
- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.
- [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
- [ ] Code generation (`make codegen`) has been run to update object specifications, if necessary.

### CODE REVIEW

**Review by leseb — COMMENTED** [review]
What is this fixing?

File: deploy/examples/create-external-cluster-resources.py [hunk 1]
```diff
@@ -72,7 +72,7 @@ def __init__(self):
         self.dummy_host_ip_map = {}
 
     def _init_cmd_output_map(self):
-        json_file_name = 'test-data/ceph-status-out'
+        json_file_name = 'tests/ceph-status-out'
```
Comment (parth-gr) [inline deploy/examples/create-external-cluster-resources.py:75]: The directory is been also changed, 

so I guess we need to visit the outer directory first something like this, `../../tests/ceph-status-out`

**Comment by subhamkrai** [comment]
> What is this fixing?

with your pr we moved file from  `cluster/examples/kubernetes/ceph/test-data/ceph-status-out` to `tests/ceph-status-out` dir.

**Comment by subhamkrai** [comment]
I was wrong it is not required. Thanks!

**Pull request closed without merging**